### DISCUSSION TITLE
Repository browser URL markup

### DISCUSSION THREAD

**Discussion by rffontenelle** [WeblateOrg/weblate#11662] [Q&A]
Is it possible to use templates markup other than `{{ branch }}`, `{{ filename }}` and `{{ line }}` ?

The [Repository browser](https://docs.weblate.org/en/weblate-5.5.5/admin/projects.html#component-repoweb) doc mentions the [Template markup](https://docs.weblate.org/en/weblate-5.5.5/admin/projects.html#markup) doc, which lists `{{ component_name }}` among other templates.  I don't see documented an explicit restriction of what templates can be used in repository browser, but I'm trying to use `{{ component_name }}` and I'm getting "Enter a valid URL." error message instead.

**Comment by nijel** [comment]
It seems like it no longer behaves as intended, created https://github.com/WeblateOrg/weblate/issues/11669 to track this.
